AUTHORS

```diff
@@ -1,2 +1,3 @@
+Dropbox, Inc.
```

Comment:
Can you add Dropbox, Inc. to AUTHORS in a separate PR? The change is valid, but it is weird to do it inside a bug fix PR.